app/src/main/java/com/rentminder/dto/Household.kt

```diff
@@ -1,7 +1,7 @@
 package com.rentminder.dto
 
-data class Household (var householdId : Int = 0, var houseName : String, var houseMembers : Int){
+data class Household (var householdId : Int = 0, var householdName : String, var householdMembers : Int){
```

Comment:
Changed two variables here to 'householdName', and 'householdMembers'. This was to make the variables both more descriptive, and also more consistent with the first variable 'householdID'.

Reply:
Since the class is already names Household, appending Household to the attributes is redundant.  I'd remove that entirely, and just name these "name" and "members".